Scheduler/observing_blocks_maintel/BLOCK-399.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "BLOCK-399",
+    "program": "BLOCK-399",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "MTReflector",
+                "cmd": "open"
+            }
+        },
+        {
+            "name": "maintel/make_lsstcam_calibrations.py",
+            "standard": false,
+            "parameters": {
+                "script_mode": "BIAS_DARK",
+                "n_bias": 3,
+                "n_discard_bias": 1,
+                "n_dark": 3,
+                "n_discard_dark": 1,
+                "exp_times_dark": [15, 30, 38],
+                "wait_between_exposures": 30,
+                "generate_calibrations": true,
+                "do_verify": true
+            }
+        },
+        {
+            "name": "maintel/setup_whitelight_flats.py",
+            "standard": false,
+            "parameters":{
+                "ignore": "$ignore"
+            }
+        },
+        {
+            "name": "maintel/take_whitelight_flats_lsstcam.py",
+            "standard": false,
+            "parameters": {
+                "sequence_names": "$sequence_names",
+                "reason": "flat",
```

Comment:
Since these exposures will already have img_type = FLAT, I wonder if it would be more useful to use `"reason": "daily_flat"` here for something more descriptive.